azure-pipelines.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       name: $(pool.name)
 
     variables:
-      - template: ../variables/globals.yml
+      - template: ./eng/pipelines/templates/variables/globals.yml
```

Comment:
FYI @benbp looks like this was from your earlier PR. We should make this an absolute path `/eng/pipelines/templates/variables/globals.yml`

Reply:
Not sure how this passed CI at the time. https://github.com/Azure/azure-sdk-for-go/pull/17756